src/client.cpp

```diff
@@ -819,22 +819,32 @@ static int _connect_check_vma(int ifd) {
 //------------------------------------------------------------------------------
 static int _connect_check(int ifd) {
     int rc = SOCKPERF_ERR_NONE;
+    int pollrc;
+    bool avail;
+
+#define POLL_TIMEOUT_MS 1
+
+#ifdef WIN32
     fd_set rfds, wfds;
-    struct timeval tv;
-    tv.tv_sec = 0;
-    tv.tv_usec = 500;
+    struct timeval tv = { .tv_sec = 0, .tv_usec = POLL_TIMEOUT_MS * 1000, };
```

Comment:
May be 500 instead of 1000 to be aligned with previous.

Reply:
`poll(2)` API uses milliseconds, so the lowest value is 1ms. For select(2) I would like to use the same timeout to have a single point of knowledge for these timeouts.

Also a timeout is an unlikely event in this scenario.